systers_portal/meetup/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from users import SystersUser
+from cities_light.models import City
+
+
+class MeetUpLocation(models.Model):
```

Comment:
If we name the model `MeetUpLocation`, then in the admin panel we will get `Meet up location`, because Django infers the words based on the camelcase. It would be nicer to have `Meetup Location`.